### DISCUSSION TITLE
sonos speakers in playergroup (volume and target)

### DISCUSSION THREAD

**Discussion by RMHogervorst** [music-assistant/support#3306] [Q&A] [answered]
Hi I have 2 sonos (ikea symfonisk) speakers. 

I have grouped the two speakers in music assistant with a playergroup

1. When I change the volume of a playing song it only changes the volume of one of the speakers, I would expect both to change, or that I have the ability to change them independently.
2. I no longer see the two individual speakers as players, only the player group is this expected?

**Comment by OzGav** [comment]
I will try and look at this in more depth but what version of the Sonos firmware are they running?

**Reply by RMHogervorst** [reply]
They are updated to the latest version. 
They are both at 81.1-58210. OsS2.

**Reply by RMHogervorst** [reply]
updated today to a new version of sonos (82.2-59204). Not sure if this is related, but now I do see 3 volume sliders under the playergroup and all three do what they are supposed to do, top one overall volume and the other two are labeled with the two speakers and sliding them does change the volume. 

Still can't see the speakers as individual units in the music assistant UI. (they remain available and accessible in home assistant.

**Comment by RMHogervorst** [comment]
In home assistant I do see the individual speakers and the player group as separate entities. The music assitant ui does not show the the two units as target players, only the player group

**Reply by OzGav** [reply, marked as answer]
Thtat is normal. Once you group them then the individual players disappear as they are no longer "individual"

**Comment by RMHogervorst** [comment]
I guess this solves all of my questions!
